src/utils/date.ts

```diff
@@ -120,3 +120,8 @@ export function getNextBirthday(birthday: Date): Date {
 
   return nextBirthday
 }
+
+export function isValidUnixTimestamp(data: string): boolean {
+  const timestamp = Number(data);
+  return !isNaN(timestamp) && timestamp >= 0;
+}
```

Comment:
Tiny nit: I'd probably call this parameter `value` instead of `data`

```suggestion
export function isValidUnixTimestamp(value: string): boolean {
  const timestamp = Number(value)
  return !isNaN(timestamp) && timestamp >= 0
}
```